recipe/meta.yaml

```diff
@@ -45,4 +45,5 @@ extra:
     - parthea
     - jreback
     - tswast
-    - jonparrott
```

Comment:
@jonparrott - are you OK with being removed?

Reply:
That account is no longer active. It belongs to a teammate of mine.